api/app.js

```diff
@@ -74,25 +75,6 @@ app.use(function (req, res, next) {
 });
 
 // error handler
-app.use(function (err, req, res, next) {
-  if (err instanceof createError.HttpError) {
```

Comment:
You did a great job destructuring this whole error handler. 
Can we remove the 'http-errors' module entirely now?

Reply:
We likely could, but because leaving it in isn't going to cause any problems, I decided against removing it, directly. I'm not sure of where else it is used, so I wanted to err on the side of caution.